### ISSUE TITLE
Serve Minified JS/CSS

### ISSUE THREAD

**Issue by loganfranken** [UCSantaBarbara/webguide#6]
We should serve minified JS/CSS to fit with our recommendations (http://www.ucsb.edu/webguide/resources.php).

**Comment by loganfranken** [comment]
URL of the page in the GitHub-hosted repo:
http://ucsb-wsg.github.io/resources/

**Comment by tenken** [comment]
what is package.json and what build tool are you using for this (Node, Grunt, Ember)? Can you either add a DEVELOPER_README or create and reference a wiki page explaining in a paragraph package.json and uglifyfs vs uglify-js?

**Comment by tenken** [comment]
I think it's this, but they list the dependency as uglifyjs with no hyphen for Uglify2 in their docs:
https://github.com/mishoo/UglifyJS2

**Comment by loganfranken** [comment]
Hey @tenken: Basically, I just installed UglifyJS2 as a Node dependency with Node Package Manager (npm) to do the JS minification. The `package.json` file is the package manifest for npm.

I ran the UglifyJS2 command myself for this pull request, but I was going to create a follow-up issue to add an automated build step with a README for developers as you suggest.

Let me know if you would prefer we do it all together in one pull request!
